tests/test_core.py

```diff
@@ -296,6 +296,51 @@ def test_region_processing_skip_aggregation(model_name, region_names):
     assert_iamframe_equal(obs, exp)
 
 
+@pytest.mark.parametrize(
+    "model_name, region_names",
+    [("model_a", ("region_A", "region_B")), ("model_b", ("region_A", "region_b"))],
+)
+def test_region_processing_wildcard_skip_aggregation(model_name, region_names):
```

Comment:
I think this test misses a crucial case where the code would currently still fail. The model mappings contain no aggregation instructions. I think they have to since most, if not all, real world examples contain aggregation instructions.
Because it skips all aggregation it also skips over a point where the code would still fail (I think).
I've added comments above where I think that would be.